include/jemalloc/internal/atomic.h

```diff
@@ -0,0 +1,128 @@
+#ifndef JEMALLOC_INTERNAL_ATOMIC_H
+#define JEMALLOC_INTERNAL_ATOMIC_H
+
+#if defined(JEMALLOC_ATOMIC_C_)
+#  define ATOMIC_INLINE extern inline
+#else
+#  define ATOMIC_INLINE inline
+#endif
+
+#if defined(JEMALLOC_GCC_ATOMIC_ATOMICS)
+#  include "jemalloc/internal/atomic_gcc_atomic.h"
+#elif defined(JEMALLOC_GCC_SYNC_ATOMICS)
+#  include "jemalloc/internal/atomic_gcc_sync.h"
+#elif defined(_MSC_VER)
+#  include "jemalloc/internal/atomic_msvc.h"
+#elif defined(JEMALLOC_C11_ATOMICS)
+#  include "jemalloc/internal/atomic_c11.h"
+#else
+#  error "Don't have atomics implemented on this platform."
```

Comment:
Quotes are unnecessary around ```#error``` args.  We aren't currently consistent about this, but I try to omit the quotes on new code.